src/ffi/bindings/orx_bindings.ml

```diff
@@ -1105,6 +1105,14 @@ module Bindings (F : Ctypes.FOREIGN) = struct
     let t_opt = ptr_opt T.Time_line.t
   end
 
+  module Animation = struct
```

Comment:
Hooray for more API coverage in the bindings!

Reply:
One question - for all the other modules I've tried to stay as close as possible to orx's naming. In this case that would mean calling this and all the other constructors below `Anim` rather than `Animation`. Would you be ok with making that change? Mostly for future proofing in case `Animation` becomes an upstream module later on.